worker/uniter/runner/debug/server.go

```diff
@@ -40,6 +44,9 @@ func (s *ServerSession) RunHook(hookName, charmDir string, env []string) error {
 	cmd.Env = env
 	cmd.Dir = charmDir
 	cmd.Stdin = bytes.NewBufferString(debugHooksServerScript)
+	var combinedOutput bytes.Buffer
```

Comment:
I've reworked how we get the output for the test now, can you take another look plz?